lib/src/runner/configuration/load.dart

```diff
@@ -74,6 +74,30 @@ class _ConfigurationLoader {
   Configuration _loadGlobalTestConfig() {
     var verboseTrace = _getBool("verbose_trace");
     var chainStackTraces = _getBool("chain_stack_traces");
+    var foldStackFrames = _getMap("fold_stack_frames", key: (keyNode) {
+      _validate(keyNode, "fold_stack_frames key must be a string",
+          (value) => value is String);
+      if (keyNode.value != "only" && keyNode.value != "except") {
+        throw new SourceSpanFormatException(
+            'Invalid fold_stack_frames key: '
+            'Must either be `only` or `except`.',
+            keyNode.span,
+            _source);
+      }
+      return keyNode.value;
+    }, value: (valueNode) {
+      if (valueNode is! YamlList ||
+          (valueNode as YamlList).firstWhere((value) => value is! String,
+                  orElse: () => null) !=
```

Comment:
Couldn't this `firstWhere()` call just be `any()`?

We should also verify that each element is a valid package name (which is to say, a valid Dart identifier).